src/Adapter/Country/Repository/CountryRepository.php

```diff
@@ -49,4 +50,23 @@ public function assertCountryExists(CountryId $countryId): void
             CountryNotFoundException::class
         );
     }
+
+    /**
+     * @param CountryId $countryId
+     *
+     * @return Country
+     *
+     * @throws CountryNotFoundException
+     */
+    public function get(CountryId $countryId): Country
```

Comment:
Is getByCountryId is already defined ?

Reply:
didn't get why you ask this? When reviewing, please try to be more specific about your suggestion. Its not always so intuitive to "guess" whats your point of some questions :stuck_out_tongue: 

If I got your intentions right, - see other repositories they all use get() method, its quite common practice to not be too specific for these main common methods). Idea is that its Already repository dedicated for country and the typehint also suggests already that we search by countryId, so its not worth adding that "byCountryId" suffix